test/Renci.SshNet.IntegrationTests/SshTests.cs

```diff
@@ -77,6 +78,23 @@ public void Ssh_ShellStream_Exit()
             }
         }
 
+        [TestMethod]
+        public void Ssh_CreateShellStream_WithoutPseudoTerminal()
+        {
+            using (var client = new SshClient(_connectionInfoFactory.Create()))
+            {
+                client.Connect();
+
+                using (var shellStream = client.CreateShellStream(bufferSize: 1024))
```

Comment:
This test also passes if I replace this line with `CreateShellStream("term", 80, 80, 80, 80, 1024)`. Can you make it only pass with the non-terminal-allocating version?

Reply:
Sorry I don't understand.
This is a integration test and we want to make sure "user can create a shell stream without allocating pseudo terminal and can successfully send/receive".
We can add another integration test to make sure "user can create a shell stream with pseudo terminal and can successfully send/receive", which I believe is already covered by test method `Ssh_ShellStream_Exit`
They don't affect each other, do they?

Reply:
What I mean is, presumably you have a use case for this functionality, but so far I don't see any demonstration of its use, and how its behaviour might differ to the terminal-allocating version (other than "it does not fail when the server disables PTY").

You said in the issue that this functionality can be used to exclude control sequences. Can you write a test to prove that?

Reply:
Okay I get what you mean. Will update the tests.